src/blueapi/service/runner.py

```diff
@@ -30,83 +28,81 @@ class Runner:
 
     _config: ApplicationConfig
     _subprocess: PoolClass | None
-    _initialized: bool = False
-    _error_message: str | None = None
     _use_subprocess: bool
+    _state: EnvironmentResponse
 
     def __init__(
-        self, config: ApplicationConfig | None = None, use_subprocess: bool = True
+        self,
+        config: ApplicationConfig | None = None,
+        use_subprocess: bool = True,
     ) -> None:
         self._config = config or ApplicationConfig()
         self._subprocess = None
         self._use_subprocess = use_subprocess
+        self._state = EnvironmentResponse(
+            initialized=False,
+            error_message="",
+        )
+
+    def reload(self):
+        """Reload the subprocess to account for any changes in python modules"""
+        self.stop()
+        self.start()
+        LOGGER.info("Runner reloaded")
 
     def start(self):
-        if self._subprocess is None and self._use_subprocess:
-            self._subprocess = Pool(initializer=_init_worker, processes=1)
-            self._subprocess.apply(
-                logging.basicConfig, kwds={"level": self._config.logging.level}
+        try:
```

Comment:
This does change start/stop to not be idempotent, but perhaps that doesn't matter?

Reply:
Should we be concerned about this at this stage?
Otherwise I'll probably carry on and it can be re-reviewed later.

Reply:
Unsure, I agree it should be re-reviewed later, in the context of the main PR